flow_action_components/SendHTMLEmail/force-app/main/default/classes/SendHTMLEmail.cls

```diff
@@ -83,12 +86,7 @@ public without sharing class SendHTMLEmail {
             // Optionally append the salesforce.com email signature to the email.
             // The email address of the user executing the Apex Code will be used.
             // True by default unless the user passes a value in.
-            if(curRequest.useSalesforceSignature != null) {
-                mail.setUseSignature(curRequest.useSalesforceSignature);
-            } else {
-                mail.setUseSignature(true);
-            }
-
+            mail.setUseSignature(useSalesforceSignature);
```

Comment:
let's combine these signature related lines of code in one place...

Reply:
On it - next pull